adapters/postgres/queries.go

```diff
@@ -0,0 +1,170 @@
+package postgres
```

Comment:
we can refactor the default methods to use these methods.. the operations methods that you create here receive a sql string